### PR TITLE
TUP-4404 : added a IS_VISIBLE tagged value to Property set when widge…

### PR DESCRIPTION
**Pull request by sgandon**
…t changes it's visibility.
@francisu I did set the TaggedValue of the Widget's Property when the visibility changes.
I just called is IS_VISIBLE because it is what it represents for now. I did not want to add some runtime_visibility who sound approximative to me.

### CODE REVIEW

File: daikon/src/main/java/org/talend/daikon/properties/presentation/Widget.java [hunk 1]
```diff
@@ -102,7 +102,7 @@
      */
     private int order;
 
-    private boolean visible = true;
+    private boolean hidden;
```
Comment (sgandon) [inline daikon/src/main/java/org/talend/daikon/properties/presentation/Widget.java]: I suggest you set some default value so that we know what it is. Especially here I think we should have false.

File: daikon/src/test/java/org/talend/daikon/avro/SchemaTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package org.talend.daikon.avro;
```
Comment (sgandon) [inline daikon/src/test/java/org/talend/daikon/avro/SchemaTest.java]: I don't understand why did you add an Avro schema test in this pull request. This is unrelated to the widget and property hidden.

File: daikon/src/main/java/org/talend/daikon/properties/presentation/Widget.java (hunk 1 above)
Comment (francisu) [inline daikon/src/main/java/org/talend/daikon/properties/presentation/Widget.java]: The default value in Java for something that is not initialized is either null, 0, or false. I don't believe in setting things explicitly to something they already are.

File: daikon/src/test/java/org/talend/daikon/avro/SchemaTest.java (hunk 2 above)
Comment (francisu) [inline daikon/src/test/java/org/talend/daikon/avro/SchemaTest.java]: True, it was related to something else (turned out to be a false alarm). But are you going to complain about more tests?

File: daikon/src/main/java/org/talend/daikon/properties/presentation/Widget.java (hunk 1 above)
Comment (sgandon) [inline daikon/src/main/java/org/talend/daikon/properties/presentation/Widget.java]: This way you make easier for some that read the code to understand the initial value and you also tell the the reviewer that you did no forget to set it by mistake.
